rho/scancommand.py

```diff
@@ -283,6 +283,25 @@ def _validate_options(self):
                 self.parser.print_help()
                 sys.exit(1)
 
+        # perform fact validation
+        facts = self.options.facts
+        default_facts = False
+        if facts == ['default']:
+            default_facts = True
+        elif os.path.isfile(facts[0]):
+            facts_to_collect = _read_in_file(facts[0])
+        else:
+            assert isinstance(facts, list)
+            facts_to_collect = facts
+        # check facts_to_collect is subset of utilities.DEFAULT_FACTS_TUPLE
+        all_facts = utilities.DEFAULT_FACTS_TUPLE
+        if not default_facts and not set(facts_to_collect).issubset(all_facts):
```

Comment:
The changes I made to `spit_results.py` depend on Rho never passing `facts_to_collect = ['default']`, but always expanding it into `utilities.DEFAULT_FACTS_TUPLE`. Does that still happen with this change? (Serious question.)

If it does, I would remove the `not default_facts` from this if. If the user specifies default and we don't expand `facts_to_collect` into a subset of `DEFAULT_FACT_TUPLE`, that's a bug (and also seems really unlikely).